sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/tool/adapter/mail/MailIntegrationAdapter.java

```diff
@@ -83,13 +83,15 @@ public String getName() {
     @Override
     public boolean isConnected() {
         return obtainConnectedStatus().orElseGet(() -> {
-            boolean connected = false;
+            boolean connected;
             JavaMailSenderImpl sender = (JavaMailSenderImpl) javaMailSender;
             try {
                 sender.testConnection();
                 connected = true;
             } catch (MessagingException e) {
                 // Will be thrown when SMTP not configured properly
+                LOGGER.error(e.getMessage(), e);
```

Comment:
Add cusom message please, do not pass `e.getMessage` as logger message - it will be logged anyway as part of the stacktrace